physical/cache.go

```diff
@@ -56,6 +70,7 @@ func NewCache(b Backend, size int, logger log.Logger) *Cache {
 		logger:  logger,
 		// This fails safe.
 		enabled: new(uint32),
+		noCache: pm,
```

Comment:
Should this be named something like cacheExceptions since the path manager could technically toggle the behavior?  For example, don't cache `path1/` but still cache `path1/key1` by prepending `!`.